packages/js/src/first-time-configuration/tailwind-components/steps/site-representation/user-selector.js

```diff
@@ -74,6 +74,7 @@ export default function UserSelector( { initialValue, onChangeCallback, placehol
 	}, 500 ), [] );
 
 	return <YoastComboBox
+		id="yoast-configuration-user-select"
```

Comment:
Question: I see that we don't consistently prefix our IDs in the FTC with `yoast-` (e.g. the Org/Person selector has `organization-person-select`). Actually most of the components have non-prefixed IDs
It the prefix needed here? I'd almost avoid it if not... But not a blocker